services/controller/standard/service.go

```diff
@@ -369,9 +369,11 @@ func (s *Service) epochTicker(ctx context.Context, data interface{}) {
 		"Epoch",
 		fmt.Sprintf("Prepare for epoch %d", currentEpoch+1),
 		s.chainTimeService.StartOfEpoch(currentEpoch).Add(time.Duration(offset)*time.Second),
-		s.prepareForEpoch,
-		&prepareForEpochData{
-			epoch: currentEpoch + 1,
+		func(ctx context.Context) {
+			prepData := &prepareForEpochData{
```

Comment:
Do you need to assign this here, or can you just pass the struct directly (i.e. `s.prepareForEpoch(ctx,&prepareForEpochData{...`)?